More information about replication. Fixes #346 (#349)
apache/couchdb-documentation: src/replication/replicator.rst

      }

> ``_scheduler/jobs`` shows more information such as a detailed history of

Reviewer: grammatical improvement (I know this one existed before your edit, but we may as well correct it at the same time):

```suggestion
``_scheduler/jobs`` shows more information, such as a detailed history of
```

> state changes. If a replication backed by a document has not yet started,

Reviewer: Can we use consistent terminology, and call this a "persistent replication" here, too?

```suggestion
state changes. If a persistent replication has not yet started,
```

Author: I was afraid of "introducing" a new terminology.

> has failed or is completed information about its state can only be found

Reviewer: grammatical improvements:

```suggestion
has failed, or is completed, information about its state can only be found
```

in ``_scheduler/docs``. Keep in mind that some replication documents could be
invalid and could not become a replication job. Others might be delayed

> because they are fetching say the filter code from a slow source database.

Reviewer: Improve awkward/ambiguous wording

```suggestion
because they are fetching data from a slow source database.
```
